pyuvdata/tests/test_uvdata.py

```diff
@@ -1548,3 +1548,160 @@ def test_get_feedpols():
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Single baseline
+    ant_str = '1_3'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1,3)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Single baseline with polarization
+    ant_str = '1l_3r'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3)]
+    polarizations_data = [-4]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Single baseline with single polarization in first entry
+    ant_str = '1l_3,2x_3'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3), (2, 3)]
+    polarizations_data = [-2, -4, -5, -7]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Single baseline with single polarization in last entry
+    ant_str = '1_3l,2_3x'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3),(2, 3)]
+    polarizations_data = [-2, -3, -5, -8]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Multiple baselines as list
+    ant_str = '1_2,1_3,1_10'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 2),(1, 3),(1, 10)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Multiples baselines with polarizations as list
+    ant_str = '1r_2l,1l_3l,1r_11r'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 2),(1, 3),(1, 11)]
+    polarizations_data = [-1, -2, -3]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Specific baselines with parenthesis
+    ant_str = '(1,3)_10'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 10),(3, 10)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Specific baselines with parenthesis
+    ant_str = '1_(3,10)'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3),(1, 10)]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_is_instance(polarizations,type(None))
+
+    # Antenna numbers with polarizations
+    ant_str = '(1l,2r)_(3l,6r)'
+    ant_pairs_nums,polarizations = uv.parse_ants(ant_str)
+    ant_pairs_data = [(1, 3),(1, 6),(2, 3),(2, 6)]
+    polarizations_data = [-1, -2, -3, -4]
+    nt.assert_items_equal(ant_pairs_nums,ant_pairs_data)
+    nt.assert_items_equal(polarizations,polarizations_data)
+
+    # Antenna numbers with - for avoidance
```

Comment:
Oh I think I see later that it excludes that antenna, rather than include it. Is there a more practical way to test this? The examples given aren't all that useful because e.g. `'1_10'` gives you the same result as `'1_(-3,10)'`, so why would you use the latter? Is the minus useful when otherwise using wildcards? For example, all baselines with antenna 1, excluding baselines with antenna 3?
Hmm.. after playing myself a little, it looks like `'1,-3'` is equivalent to `'1,3'`. Is this correct behavior?